tests/scripts/custodian/do-not-delete-keys.txt

```diff
@@ -0,0 +1,4 @@
+DONOTDELETE
+qahostedk3s
+qak3stenant
+DoNotDelete
```

Comment:
These keys could also be moved to an env var / param in Jenkins for updating without a commit

Reply:
resolved, removed file and will add a jenkins parameter once the jenkins job PR merges